### ISSUE TITLE
Images in docs might not appear as intended in dark mode

### ISSUE THREAD

**Issue by purva-thakre** [unitaryfund/mitiq#1547]
When RTD is displayed in dark mode, some of the images with dark borders appear cutoff. Those with white backgrounds and dark borders also appear cutoff and a little bit squashed. 

Here's a comparison for [one of the CDR images](https://mitiq.readthedocs.io/en/stable/guide/cdr-4-low-level.html). 

![image](https://user-images.githubusercontent.com/66048318/194647427-c0b925fb-0bbe-4e85-85e1-00fcf0eee29c.png) ![image](https://user-images.githubusercontent.com/66048318/194647481-09a17fc6-6498-4c75-9ea2-2a55ff5fcd8b.png)

**Comment by natestemen** [comment]
I'm not totally sure what the solution here is. Do you think the white background should be extended further around the image so the black background from dark-mode doesn't butt right up against the image?

**Comment by purva-thakre** [comment]
Yeah, I was thinking it would be better to increase the dimensions of the white background. And make this background consistent for all images in the documentation. 

There are some images without a solid white background layer as well. They appear dim in dark mode (couldn't think of a better term). 
![image](https://user-images.githubusercontent.com/66048318/196208585-68d2557a-2d07-463c-b5cb-09e6a9d8c1b0.png)
![image](https://user-images.githubusercontent.com/66048318/196208840-aed6ec6c-379c-4f3f-8762-3012a9ed0be9.png)

**Comment by jordandsullivan** [comment]
I'm not quite seeing the issue in these images? The dark mode images seem to appear fine for me. Is there information getting cut off?

**Comment by purva-thakre** [comment]
> Is there information getting cut off?

Not really. 

It's more about how there's a transparent layer underneath all images. The comparison in the description is the issue at hand. If we switch from light to dark mode, a consistent white layer underneath all images would help avoid dark mode creeping up close and behind the images.
